src/crimson/osd/pg.cc

```diff
@@ -317,7 +317,7 @@ PG::do_osd_op(ObjectState& os, OSDOp& osd_op, ceph::os::Transaction& txn)
   // TODO: dispatch via call table?
   // TODO: we might want to find a way to unify both input and output
   // of each op.
-  switch (const ceph_osd_op& op = osd_op.op; op.op) {
+  switch (ceph_osd_op& op = osd_op.op; op.op) {
```

Comment:
I think actually we shouldn't be modifying the osd_op structure, even if classic does.  It looks like it's just a convenient way of dealing with truncate_seq?

Reply:
yes. i will drop this commit. thought it could free up the memory used by `op.op`, but guess it's not a normal case. as it's only used by mds for truncating (large) files.